cmd/cdi-importer/importer.go

```diff
@@ -79,6 +87,7 @@ func main() {
 		AvailableSpace: util.GetAvailableSpace(common.ImporterVolumePath),
```

Comment:
We probably need to alter this for block devices, so we can check there is enough space on the block device.

Reply:
@awls Right now we do not resize block device which does not have filesystem. Can/Should we resize it?

Reply:
No you can't resize block devices. But you still have to check to make sure the device is large enough to hold the virtual image. For instance lets say you have 2G block device but the virtual image we are writing to it is 4G, you will end up with a corrupted disk (similar to what happened when we were resizing the image to be smaller than it originally was).

In short, no resize, but we have to check that the block device is large enough to hold the guest partitions and guest filesystems.

Reply:
@awels I changed the code. I'm running 'lsblk -bo SIZE /data/blockDevice' to find the size, and I get the following error:
I0301 13:55:50.410309       1 importer.go:124] there is an error $s fork/exec lsblk -bo SIZE /data/blockDevice: no such file or directory

When I get into the pod, and run this command, it returns the size (~1G as the PV size).
Also the import is successful to /data/blockDevice

So the question is how come it does not find /data/blockDevice if it imports into it a sec after?
Is the command signature wrong?
I wonder if you or others know.....